ui/src/helpers/defaults-helpers.ts

```diff
@@ -82,3 +82,61 @@ export const getPropertyPaths = (
   // For primitive types and other unsupported types, return an empty array
   return [];
 };
+
+/**
+ * Encode the property path based on the locked status.
```

Comment:
What *is* a "lock status" and what is it required for?

Reply:
Note to myself: "Locked" default values are defaults that cannot be overridden, as they will cause the corresponding run form field to become disabled.